### PR TITLE
[box]Adds a 2% xeno egg spawner to xenobio

### PR DESCRIPTION
**Pull request by patpol4**
### Intent of your Pull Request

title,
The thing exists as on object so I figure we might as well use it.
Mainly making this to see if people think this is a horrible or a good idea

it's just 2%, remember

### Why is this change good for the game?
Variety is the spice of life
also makes for some interesting science action

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
title

### What general grouping does this PR fall under? 
xenobio

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
uuh
### If there are any numerical values involved in your PR that will be relevant to a player, 2% spawn chance

# Changelog

:cl:  
rsctweak: Adds a 2% xeno egg spawner to xenobio in boxstation
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
Probably a horrible idea, but hey why not?

**Comment by TheLinuxOS** [comment]
Station disaster waiting to happen.... I absolutely love it.

**Comment by ToasterBiome** [comment]
I actually thought there used to be a 2% xeno egg in there
